interface/patient_tracker/patient_tracker.php

```diff
@@ -175,263 +176,197 @@
-                                    <span class="font-weight-bold" style="text-decoration:underline; font-size:1.2em;">MedEx <?php echo xlt('Reminders'); ?></span><br/>
-                                    <div class="text-left blockquote" style="width: 65%; margin: 5px auto;">
-                                        <a href="https://medexbank.com/cart/upload/index.php?route=information/campaigns&amp;g=rem"
-                                           target="_medex">
-                                            <?php echo $current_events; ?>
-                                        </a>
+                                </div>
+                                <?php
+                                if ($GLOBALS['medex_enable'] == '1') {
+                                    ?>
+                                <div class="col-sm-<?php echo attr($col_width) . " " . attr($last_col_width); ?> text-center">
+                                    <div class="text-center">
+                                        <span class="font-weight-bold" style="text-decoration:underline; font-size:1.2em;">MedEx <?php echo xlt('Reminders'); ?></span><br/>
+                                        <div class="text-left blockquote" style="width: 65%; margin: 5px auto;">
+                                            <a href="https://medexbank.com/cart/upload/index.php?route=information/campaigns&amp;g=rem"
+                                                target="_medex">
+                                                <?php echo $current_events; ?>
+                                            </a>
+                                        </div>
                                     </div>
                                 </div>
-                        </div>
-                        <?php } ?>
+                                <?php } ?>
+                            </div>
+                            <div id="message" class="warning"></div>
+                        </form>
                     </div>
-                <div id="message" class="warning"></div>
+                </div>
             </div>
-            </form>
-        </div>
-    </div>
 
     <div class="row-fluid">
         <div class="col-md-12">
-            <div class="text-center row divTable" style="width: 85%; padding: 10px 10px 0; margin: 10px auto;">
+            <div class="text-center row mx-auto divTable">
                 <div class="col-sm-12" id="loader">
-                    <div class="text-center">
-                        <i class="fa fa-spinner fa-pulse fa-fw" style="font-size: 140px; color: var(--gray700); padding: 20px"></i>
-                        <h2><?php echo xlt('Loading data'); ?>...</h2>
+                    <div class="spinner-border" role="status">
+                        <span class="sr-only">Loading...</span>
                     </div>
+                    <h2><?php echo xlt('Loading data'); ?>...</h2>
                 </div>
```

Comment:
@tywrenn 
I moved it into the line 363.

Reply:
Yeah but you also have to do the text in the "sr-only" class span

Reply:
OK, Update commit :)